tracer/src/Datadog.Trace.SourceGenerators/TagsListGenerator/Sources.cs

```diff
@@ -80,27 +82,61 @@ partial class ")
             {
                 foreach (var property in tagList.MetricProperties)
                 {
+                    var tagBytes = Encoding.UTF8.GetBytes(property.TagValue);
+                    var tagArray = new string[tagBytes.Length];
+                    for (var i = 0; i < tagBytes.Length; i++)
+                    {
+                        tagArray[i] = tagBytes[i].ToString();
+                    }
+
+                    var tagByteArray = "{ " + string.Join(", ", tagArray) + " }";
```

Comment:
It's nit-picking, but you can avoid the string concatenation here if you combine with the suggestion below:
```suggestion
                    var tagByteArray = string.Join(", ", tagArray);
```

Reply:
Oh I missed that, you're right, done in https://github.com/DataDog/dd-trace-dotnet/pull/2753/commits/9d2c4e8620171847cbe0aa374120ab261ad7eb8d